proto/bitcoin/bitcoind/v1alpha/bitcoin.proto

```diff
@@ -303,6 +304,21 @@ message SendResponse {
   RawTransaction tx = 3;
 }
 
+message SendToAddressRequest {
```

Comment:
Should add `string wallet` here

Reply:
That doesnt exist on that rpc

Reply:
Or am I misunderstanding completely how the wallet stuff works?

Reply:
yes you are misunderstanding! if you look at the other commands that use `rpcForWallet` you'll see that none of them take a `wallet` parameter. However, they all interact with a specific wallet,  which is kinda of an implicit parameter. It's equivalent to `-rpcwallet` in `bitcoin-cli`

Reply:
almost like a header